src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -112,7 +114,64 @@ runnable example.
 
 ## Enable OTLP Exporter for all signals
 
-Content coming soon.
+A cross-cutting extension was added in v1.8.0 to simplify registration of the
```

Comment:
Since we haven't released this package yet, I think a better way to put this could be something like this:
```suggestion
Starting `1.8.0` version, you can use the cross-cutting extension to simplify registration of the
```

Reply:
I pushed a tweak